### ISSUE TITLE
Issue getting the current day again

### ISSUE THREAD

**Issue by hanj0415** [kevinjalbert/notion-toolbox#19]
Referred to #6, and modified get_current_day.py however I have below logs from Alfred and shows cannot execute the program.

```
[18:51:26.467] Notion[Keyword] Processing complete
[18:51:26.468] Notion[Keyword] Passing output '' to Run Script
[18:51:29.138] Notion[Run Script] Processing complete
[18:51:29.143] Notion[Run Script] Passing output '[
  <BreadcrumbBlock (id='36f97810-d9b0-426d-b5df-d7ed567ca851')>,
  <ColumnListBlock (id='0c954592-6ff4-4493-a0a8-94a1e4a9f719')>,
  <Block (id='fcc601a0-1e67-4c6d-9f1b-02badfe66e18', type='copy_indicator')>,
  <Block (id='03495dbb-8a19-4816-90be-be0b4e1d93c0', type='copy_indicator')>,
  <Block (id='4e752f33-e04a-481a-9fb1-2fe21f1a6464', type='copy_indicator')>,
  <Block (id='c53bbad0-d99b-4909-9189-a6d7738bfee6', type='copy_indicator')>,
  <CollectionViewBlock (id='a40f3028-dcf1-418e-a5e4-41618d60299f', title='🚦 2020 > Week 24 >  Lights', collection=<Collection (id='7917e894-6f7a-4936-b08a-54ae201bbf03')>)>,
  <TextBlock (id='c9e83947-0bc3-4eb5-b51f-ad2c8834e5bb', title="⬇️ Set table's filter to current week")>,
  <CollectionViewBlock (id='7de4e382-08cb-4c47-927a-c7604d0d46d9', title='🏆 Wins', collection=<Collection (id='305707c2-fef8-469e-b393-017bc290e72f')>)>,
  <SubheaderBlock (id='c7c4f972-f7fe-470a-a01e-fe66931d1509', title='Weekly Review')>,
  <SubsubheaderBlock (id='974d286c-e64b-45b6-b4d6-34ed0a416c9a', title='Lights (Aim for 5-6 *Yes,* adjust if outside of this)')>,
  <BulletedListBlock (id='84e5a9c2-3a85-4f12-ad98-6bf9393e9770', title='_Reflection_on_objectives_what_changes_for_next_week_')>,
  <SubsubheaderBlock (id='19a2e757-29ce-4cb6-985a-6348c583c143', title="RescueTime (make link to week's dashboard)")>,
  <TextBlock (id='c934e2af-6781-4ff6-accb-52f6805c2470', title='_Screenshot_of_dashboard_')>,
  <TextBlock (id='6c49a3c2-c584-4e69-aef1-7d0dda2a2c2f')>,
]
[
  <ColumnBlock (id='13a29fcd-c67a-4d4d-992c-07cc8177aad1')>,
  <ColumnBlock (id='361d528f-f6a0-4ef7-ad2a-8077827a5ce5')>,
]
[
  <SubheaderBlock (id='e4cc781c-c5a8-4e04-81ba-dca1d38e573f', title='Days')>,
  <PageBlock (id='936e658c-87ba-40df-977d-0b7795fd698f', title='June 7 (Sunday)')>,
  <PageBlock (id='7062af69-f86a-4c21-ad41-1cacccae6a3b', title='June 8  (Monday)')>,
  <PageBlock (id='21929fc5-77ad-42de-b0a0-d2b6e9256643', title='June 9 (Tuesday)')>,
  <PageBlock (id='73246907-752c-4e08-a895-8be9d2efb612', title='June 10 (Wednesday)')>,
  <PageBlock (id='1a445d6b-8fd2-420f-bc10-076c7de999b5', title='June 11 (Thursday)')>,
  <PageBlock (id='36b55dbb-0455-4cc4-9876-8866c6af98e4', title='June 12 (Friday)')>,
  <PageBlock (id='6bc70839-b6b0-47b7-a45a-0101f1a0229f', title='June 13 (Saturday)')>,
]
notion://www.notion.so/36b55dbb04554cc498768866c6af98e4
' to Filter
[18:51:29.160] Notion[Filter] Processing complete
[18:51:29.160] Notion[Filter] Passing output '[
  <BreadcrumbBlock (id='36f97810-d9b0-426d-b5df-d7ed567ca851')>,
  <ColumnListBlock (id='0c954592-6ff4-4493-a0a8-94a1e4a9f719')>,
  <Block (id='fcc601a0-1e67-4c6d-9f1b-02badfe66e18', type='copy_indicator')>,
  <Block (id='03495dbb-8a19-4816-90be-be0b4e1d93c0', type='copy_indicator')>,
  <Block (id='4e752f33-e04a-481a-9fb1-2fe21f1a6464', type='copy_indicator')>,
  <Block (id='c53bbad0-d99b-4909-9189-a6d7738bfee6', type='copy_indicator')>,
  <CollectionViewBlock (id='a40f3028-dcf1-418e-a5e4-41618d60299f', title='🚦 2020 > Week 24 >  Lights', collection=<Collection (id='7917e894-6f7a-4936-b08a-54ae201bbf03')>)>,
  <TextBlock (id='c9e83947-0bc3-4eb5-b51f-ad2c8834e5bb', title="⬇️ Set table's filter to current week")>,
  <CollectionViewBlock (id='7de4e382-08cb-4c47-927a-c7604d0d46d9', title='🏆 Wins', collection=<Collection (id='305707c2-fef8-469e-b393-017bc290e72f')>)>,
  <SubheaderBlock (id='c7c4f972-f7fe-470a-a01e-fe66931d1509', title='Weekly Review')>,
  <SubsubheaderBlock (id='974d286c-e64b-45b6-b4d6-34ed0a416c9a', title='Lights (Aim for 5-6 *Yes,* adjust if outside of this)')>,
  <BulletedListBlock (id='84e5a9c2-3a85-4f12-ad98-6bf9393e9770', title='_Reflection_on_objectives_what_changes_for_next_week_')>,
  <SubsubheaderBlock (id='19a2e757-29ce-4cb6-985a-6348c583c143', title="RescueTime (make link to week's dashboard)")>,
  <TextBlock (id='c934e2af-6781-4ff6-accb-52f6805c2470', title='_Screenshot_of_dashboard_')>,
  <TextBlock (id='6c49a3c2-c584-4e69-aef1-7d0dda2a2c2f')>,
]
[
  <ColumnBlock (id='13a29fcd-c67a-4d4d-992c-07cc8177aad1')>,
  <ColumnBlock (id='361d528f-f6a0-4ef7-ad2a-8077827a5ce5')>,
]
[
  <SubheaderBlock (id='e4cc781c-c5a8-4e04-81ba-dca1d38e573f', title='Days')>,
  <PageBlock (id='936e658c-87ba-40df-977d-0b7795fd698f', title='June 7 (Sunday)')>,
  <PageBlock (id='7062af69-f86a-4c21-ad41-1cacccae6a3b', title='June 8  (Monday)')>,
  <PageBlock (id='21929fc5-77ad-42de-b0a0-d2b6e9256643', title='June 9 (Tuesday)')>,
  <PageBlock (id='73246907-752c-4e08-a895-8be9d2efb612', title='June 10 (Wednesday)')>,
  <PageBlock (id='1a445d6b-8fd2-420f-bc10-076c7de999b5', title='June 11 (Thursday)')>,
  <PageBlock (id='36b55dbb-0455-4cc4-9876-8866c6af98e4', title='June 12 (Friday)')>,
  <PageBlock (id='6bc70839-b6b0-47b7-a45a-0101f1a0229f', title='June 13 (Saturday)')>,
]
notion://www.notion.so/36b55dbb04554cc498768866c6af98e4
' to Transform
[18:51:29.163] Notion[Transform] Processing complete
[18:51:29.164] Notion[Transform] Passing output '[
  <BreadcrumbBlock (id='36f97810-d9b0-426d-b5df-d7ed567ca851')>,
  <ColumnListBlock (id='0c954592-6ff4-4493-a0a8-94a1e4a9f719')>,
  <Block (id='fcc601a0-1e67-4c6d-9f1b-02badfe66e18', type='copy_indicator')>,
  <Block (id='03495dbb-8a19-4816-90be-be0b4e1d93c0', type='copy_indicator')>,
  <Block (id='4e752f33-e04a-481a-9fb1-2fe21f1a6464', type='copy_indicator')>,
  <Block (id='c53bbad0-d99b-4909-9189-a6d7738bfee6', type='copy_indicator')>,
  <CollectionViewBlock (id='a40f3028-dcf1-418e-a5e4-41618d60299f', title='🚦 2020 > Week 24 >  Lights', collection=<Collection (id='7917e894-6f7a-4936-b08a-54ae201bbf03')>)>,
  <TextBlock (id='c9e83947-0bc3-4eb5-b51f-ad2c8834e5bb', title="⬇️ Set table's filter to current week")>,
  <CollectionViewBlock (id='7de4e382-08cb-4c47-927a-c7604d0d46d9', title='🏆 Wins', collection=<Collection (id='305707c2-fef8-469e-b393-017bc290e72f')>)>,
  <SubheaderBlock (id='c7c4f972-f7fe-470a-a01e-fe66931d1509', title='Weekly Review')>,
  <SubsubheaderBlock (id='974d286c-e64b-45b6-b4d6-34ed0a416c9a', title='Lights (Aim for 5-6 *Yes,* adjust if outside of this)')>,
  <BulletedListBlock (id='84e5a9c2-3a85-4f12-ad98-6bf9393e9770', title='_Reflection_on_objectives_what_changes_for_next_week_')>,
  <SubsubheaderBlock (id='19a2e757-29ce-4cb6-985a-6348c583c143', title="RescueTime (make link to week's dashboard)")>,
  <TextBlock (id='c934e2af-6781-4ff6-accb-52f6805c2470', title='_Screenshot_of_dashboard_')>,
  <TextBlock (id='6c49a3c2-c584-4e69-aef1-7d0dda2a2c2f')>,
]
[
  <ColumnBlock (id='13a29fcd-c67a-4d4d-992c-07cc8177aad1')>,
  <ColumnBlock (id='361d528f-f6a0-4ef7-ad2a-8077827a5ce5')>,
]
[
  <SubheaderBlock (id='e4cc781c-c5a8-4e04-81ba-dca1d38e573f', title='Days')>,
  <PageBlock (id='936e658c-87ba-40df-977d-0b7795fd698f', title='June 7 (Sunday)')>,
  <PageBlock (id='7062af69-f86a-4c21-ad41-1cacccae6a3b', title='June 8  (Monday)')>,
  <PageBlock (id='21929fc5-77ad-42de-b0a0-d2b6e9256643', title='June 9 (Tuesday)')>,
  <PageBlock (id='73246907-752c-4e08-a895-8be9d2efb612', title='June 10 (Wednesday)')>,
  <PageBlock (id='1a445d6b-8fd2-420f-bc10-076c7de999b5', title='June 11 (Thursday)')>,
  <PageBlock (id='36b55dbb-0455-4cc4-9876-8866c6af98e4', title='June 12 (Friday)')>,
  <PageBlock (id='6bc70839-b6b0-47b7-a45a-0101f1a0229f', title='June 13 (Saturday)')>,
]
notion://www.notion.so/36b55dbb04554cc498768866c6af98e4' to Open URL
[18:51:29.168] Notion[Run Script] Passing output '[
  <BreadcrumbBlock (id='36f97810-d9b0-426d-b5df-d7ed567ca851')>,
  <ColumnListBlock (id='0c954592-6ff4-4493-a0a8-94a1e4a9f719')>,
  <Block (id='fcc601a0-1e67-4c6d-9f1b-02badfe66e18', type='copy_indicator')>,
  <Block (id='03495dbb-8a19-4816-90be-be0b4e1d93c0', type='copy_indicator')>,
  <Block (id='4e752f33-e04a-481a-9fb1-2fe21f1a6464', type='copy_indicator')>,
  <Block (id='c53bbad0-d99b-4909-9189-a6d7738bfee6', type='copy_indicator')>,
  <CollectionViewBlock (id='a40f3028-dcf1-418e-a5e4-41618d60299f', title='🚦 2020 > Week 24 >  Lights', collection=<Collection (id='7917e894-6f7a-4936-b08a-54ae201bbf03')>)>,
  <TextBlock (id='c9e83947-0bc3-4eb5-b51f-ad2c8834e5bb', title="⬇️ Set table's filter to current week")>,
  <CollectionViewBlock (id='7de4e382-08cb-4c47-927a-c7604d0d46d9', title='🏆 Wins', collection=<Collection (id='305707c2-fef8-469e-b393-017bc290e72f')>)>,
  <SubheaderBlock (id='c7c4f972-f7fe-470a-a01e-fe66931d1509', title='Weekly Review')>,
  <SubsubheaderBlock (id='974d286c-e64b-45b6-b4d6-34ed0a416c9a', title='Lights (Aim for 5-6 *Yes,* adjust if outside of this)')>,
  <BulletedListBlock (id='84e5a9c2-3a85-4f12-ad98-6bf9393e9770', title='_Reflection_on_objectives_what_changes_for_next_week_')>,
  <SubsubheaderBlock (id='19a2e757-29ce-4cb6-985a-6348c583c143', title="RescueTime (make link to week's dashboard)")>,
  <TextBlock (id='c934e2af-6781-4ff6-accb-52f6805c2470', title='_Screenshot_of_dashboard_')>,
  <TextBlock (id='6c49a3c2-c584-4e69-aef1-7d0dda2a2c2f')>,
]
[
  <ColumnBlock (id='13a29fcd-c67a-4d4d-992c-07cc8177aad1')>,
  <ColumnBlock (id='361d528f-f6a0-4ef7-ad2a-8077827a5ce5')>,
]
[
  <SubheaderBlock (id='e4cc781c-c5a8-4e04-81ba-dca1d38e573f', title='Days')>,
  <PageBlock (id='936e658c-87ba-40df-977d-0b7795fd698f', title='June 7 (Sunday)')>,
  <PageBlock (id='7062af69-f86a-4c21-ad41-1cacccae6a3b', title='June 8  (Monday)')>,
  <PageBlock (id='21929fc5-77ad-42de-b0a0-d2b6e9256643', title='June 9 (Tuesday)')>,
  <PageBlock (id='73246907-752c-4e08-a895-8be9d2efb612', title='June 10 (Wednesday)')>,
  <PageBlock (id='1a445d6b-8fd2-420f-bc10-076c7de999b5', title='June 11 (Thursday)')>,
  <PageBlock (id='36b55dbb-0455-4cc4-9876-8866c6af98e4', title='June 12 (Friday)')>,
  <PageBlock (id='6bc70839-b6b0-47b7-a45a-0101f1a0229f', title='June 13 (Saturday)')>,
]
notion://www.notion.so/36b55dbb04554cc498768866c6af98e4
' to Call External Trigger
[18:51:29.173] Notion[External] Processing complete
[18:51:29.173] Notion[External] Passing output '[
  <BreadcrumbBlock (id='36f97810-d9b0-426d-b5df-d7ed567ca851')>,
  <ColumnListBlock (id='0c954592-6ff4-4493-a0a8-94a1e4a9f719')>,
  <Block (id='fcc601a0-1e67-4c6d-9f1b-02badfe66e18', type='copy_indicator')>,
  <Block (id='03495dbb-8a19-4816-90be-be0b4e1d93c0', type='copy_indicator')>,
  <Block (id='4e752f33-e04a-481a-9fb1-2fe21f1a6464', type='copy_indicator')>,
  <Block (id='c53bbad0-d99b-4909-9189-a6d7738bfee6', type='copy_indicator')>,
  <CollectionViewBlock (id='a40f3028-dcf1-418e-a5e4-41618d60299f', title='🚦 2020 > Week 24 >  Lights', collection=<Collection (id='7917e894-6f7a-4936-b08a-54ae201bbf03')>)>,
  <TextBlock (id='c9e83947-0bc3-4eb5-b51f-ad2c8834e5bb', title="⬇️ Set table's filter to current week")>,
  <CollectionViewBlock (id='7de4e382-08cb-4c47-927a-c7604d0d46d9', title='🏆 Wins', collection=<Collection (id='305707c2-fef8-469e-b393-017bc290e72f')>)>,
  <SubheaderBlock (id='c7c4f972-f7fe-470a-a01e-fe66931d1509', title='Weekly Review')>,
  <SubsubheaderBlock (id='974d286c-e64b-45b6-b4d6-34ed0a416c9a', title='Lights (Aim for 5-6 *Yes,* adjust if outside of this)')>,
  <BulletedListBlock (id='84e5a9c2-3a85-4f12-ad98-6bf9393e9770', title='_Reflection_on_objectives_what_changes_for_next_week_')>,
  <SubsubheaderBlock (id='19a2e757-29ce-4cb6-985a-6348c583c143', title="RescueTime (make link to week's dashboard)")>,
  <TextBlock (id='c934e2af-6781-4ff6-accb-52f6805c2470', title='_Screenshot_of_dashboard_')>,
  <TextBlock (id='6c49a3c2-c584-4e69-aef1-7d0dda2a2c2f')>,
]
[
  <ColumnBlock (id='13a29fcd-c67a-4d4d-992c-07cc8177aad1')>,
  <ColumnBlock (id='361d528f-f6a0-4ef7-ad2a-8077827a5ce5')>,
]
[
  <SubheaderBlock (id='e4cc781c-c5a8-4e04-81ba-dca1d38e573f', title='Days')>,
  <PageBlock (id='936e658c-87ba-40df-977d-0b7795fd698f', title='June 7 (Sunday)')>,
  <PageBlock (id='7062af69-f86a-4c21-ad41-1cacccae6a3b', title='June 8  (Monday)')>,
  <PageBlock (id='21929fc5-77ad-42de-b0a0-d2b6e9256643', title='June 9 (Tuesday)')>,
  <PageBlock (id='73246907-752c-4e08-a895-8be9d2efb612', title='June 10 (Wednesday)')>,
  <PageBlock (id='1a445d6b-8fd2-420f-bc10-076c7de999b5', title='June 11 (Thursday)')>,
  <PageBlock (id='36b55dbb-0455-4cc4-9876-8866c6af98e4', title='June 12 (Friday)')>,
  <PageBlock (id='6bc70839-b6b0-47b7-a45a-0101f1a0229f', title='June 13 (Saturday)')>,
]
notion://www.notion.so/36b55dbb04554cc498768866c6af98e4
' to Filter
```

It seems it is passing the entire outputs as a variable, not the browseable URL only.

**Comment by kevinjalbert** [comment]
@hanj0415 can you double-check this after we have #18 resolved? That issue seems to be related to an issue in the day formatting option, which would affect how fetching the current day would work.

**Comment by kevinjalbert** [comment]
@hanj0415 give this a shot again with the new version which has the fix from #21. I would recommend keeping the new optional (for day format and week start day) configs out unless you are planning to use them.

**Comment by hanj0415** [comment]
@kevinjalbert I've removed ```WEEK_STARTS_ON_SUNDAY``` and ```CUSTOM_DAY_FORMAT``` from the config. And I have the same result with the error "Cannot execute the program (-50)". I tried to execute get_current_day.py, and I had below.
```
❯ python3 get_current_day.py
[
  <BreadcrumbBlock (id='36f97810-d9b0-426d-b5df-d7ed567ca851')>,
  <ColumnListBlock (id='0c954592-6ff4-4493-a0a8-94a1e4a9f719')>,
  <Block (id='fcc601a0-1e67-4c6d-9f1b-02badfe66e18', type='copy_indicator')>,
  <Block (id='03495dbb-8a19-4816-90be-be0b4e1d93c0', type='copy_indicator')>,
  <Block (id='4e752f33-e04a-481a-9fb1-2fe21f1a6464', type='copy_indicator')>,
  <Block (id='c53bbad0-d99b-4909-9189-a6d7738bfee6', type='copy_indicator')>,
  <CollectionViewBlock (id='a40f3028-dcf1-418e-a5e4-41618d60299f', title='🚦 2020 > Week 24 >  Lights', collection=<Collection (id='7917e894-6f7a-4936-b08a-54ae201bbf03')>)>,
  <TextBlock (id='c9e83947-0bc3-4eb5-b51f-ad2c8834e5bb', title="⬇️ Set table's filter to current week")>,
  <CollectionViewBlock (id='7de4e382-08cb-4c47-927a-c7604d0d46d9', title='🏆 Wins', collection=<Collection (id='305707c2-fef8-469e-b393-017bc290e72f')>)>,
  <SubheaderBlock (id='c7c4f972-f7fe-470a-a01e-fe66931d1509', title='Weekly Review')>,
  <SubsubheaderBlock (id='974d286c-e64b-45b6-b4d6-34ed0a416c9a', title='Lights (Aim for 5-6 *Yes,* adjust if outside of this)')>,
  <BulletedListBlock (id='84e5a9c2-3a85-4f12-ad98-6bf9393e9770', title='_Reflection_on_objectives_what_changes_for_next_week_')>,
  <SubsubheaderBlock (id='19a2e757-29ce-4cb6-985a-6348c583c143', title="RescueTime (make link to week's dashboard)")>,
  <TextBlock (id='c934e2af-6781-4ff6-accb-52f6805c2470', title='_Screenshot_of_dashboard_')>,
  <TextBlock (id='6c49a3c2-c584-4e69-aef1-7d0dda2a2c2f')>,
]
[
  <ColumnBlock (id='13a29fcd-c67a-4d4d-992c-07cc8177aad1')>,
  <ColumnBlock (id='361d528f-f6a0-4ef7-ad2a-8077827a5ce5')>,
]
[
  <SubheaderBlock (id='e4cc781c-c5a8-4e04-81ba-dca1d38e573f', title='Days')>,
  <PageBlock (id='936e658c-87ba-40df-977d-0b7795fd698f', title='June 7 (Sunday)')>,
  <PageBlock (id='7062af69-f86a-4c21-ad41-1cacccae6a3b', title='June 8 (Monday)')>,
  <PageBlock (id='21929fc5-77ad-42de-b0a0-d2b6e9256643', title='June 9 (Tuesday)')>,
  <PageBlock (id='73246907-752c-4e08-a895-8be9d2efb612', title='June 10 (Wednesday)')>,
  <PageBlock (id='1a445d6b-8fd2-420f-bc10-076c7de999b5', title='June 11 (Thursday)')>,
  <PageBlock (id='36b55dbb-0455-4cc4-9876-8866c6af98e4', title='June 12 (Friday)')>,
  <PageBlock (id='6bc70839-b6b0-47b7-a45a-0101f1a0229f', title='June 13 (Saturday)')>,
]
notion://www.notion.so/6bc70839b6b047b7a45a0101f1a0229f
```

**Comment by hanj0415** [comment]
@kevinjalbert I don't know why but it worked back. I redownloaded everything. And It just worked. I don't know why but I have a proper output.
I want to point out that notion-py has a bug in the setup.py with new PIP; https://github.com/xnutsive/notion-py/tree/bugfix/pip-install-fail. And The default path for python 3 has been changed to /usr/bin/ from /usr/local/bin in Catalina. I made a symbolic link.
